spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -781,6 +781,162 @@
       end
     end
 
+    describe "line 27 - total income" do
+      context "when taxpayer provides total income with the sum 200,001.33" do
+        let(:submission) {
+          create :efile_submission, tax_return: nil, data_source: create(
+            :state_file_nj_intake,
+            :df_data_many_w2s
+          )
+        }
+        it "fills in the total income boxes in the PDF on line 27 with the rounded value" do
+          # millions
+          expect(pdf_fields["263"]).to eq ""
```

Comment:
[dust] I wonder if we should make a utility similar to the `insert_digits` one in the filler to help test pdf fields with digits?

Reply:
love this idea